deposit/src/main/java/edu/unc/lib/deposit/fcrepo4/IngestContentObjectsJob.java

```diff
@@ -211,6 +211,8 @@ public void runJob() {
         } catch (DepositException | FedoraException | IOException e) {
             failJob(e, "Failed to ingest content for deposit {0}", getDepositPID().getQualifiedId());
         }
+
+        // TODO verify objects present in fcrepo
```

Comment:
Is there a ticket created for this future work?

Reply:
created
https://jira.lib.unc.edu/browse/CDR-2006